tests/Pest/FormattingTest.php

```diff
@@ -45,3 +46,13 @@
     expect(Money::fromDecimal(10.45)->formatted(['decimalSeparator' => ',', 'prefix' => '$$$']))->toBe('$$$10,45');
     expect(Money::fromDecimal(10.45)->formatted(decimalSeparator: ',', suffix: ' USD'))->toBe('$10,45 USD');
 });
+
+test('setting for deleting trailing decimal zeros', function () {
```

Comment:
This should follow our test naming format — actually describe what's being asserted